backend/app/scripts/init_db.py

```
@@ -0,0 +1,25 @@
+def main():
+    logger.info(f"Connecting to FalkorDB at {settings.falkordb_url}")
+
+    try:
+        # FalkorDB.from_url typically parses redis:// URLs
+        # In the python falkordb package, the typical initialization is:
+        # db = FalkorDB(host='...', port=...)
+        # We can use from_url since falkordb wraps redis
+        client = FalkorDB.from_url(settings.falkordb_url)
+        init_schema(client)
+        logger.info("Successfully initialized FalkorDB schema.")
+    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)